examples/directoryNavigationDemo.html

```diff
@@ -272,32 +279,6 @@ <h2>Groups</h2>
                 }
             }
 
-            /**
-             * Determine the embedding Genesys Cloud environment seeded on the query string or
-             * being returned through the OAuth2 Implicit grant state hash param.
-             *
-             * @returns A string indicating the embedding PC env (e.g. mypurecloud.com, mypurecloud.jp); otherwise, null.
-             */
-            function getEmbeddingPCEnv() {
```

Comment:
thanks for cleaning up all these duplicated utils.  Years of tech debt in the making.